install/gcp/installer/util/utils.py

```diff
@@ -106,6 +106,37 @@ def _print_banner(border_symbol, edge_symbol, corner_symbol,
     print(border)
     print('')
 
+def get_latest_patch_query():
```

Comment:
The naming of this method can be better, to reflect what it is doing.  I suggest:
```
def get_all_patches_glob():
def get_all_patches_glob_in_current_version():
def get_glob_for_all_patches_in_current_version():
```
#1: you are finding all patches, not just the latest patch
#2: `glob` is more clear than `query`, it matches the documentation and when we discuss verbally